nodes/mainnet/src/lib.rs

```diff
@@ -22,3 +22,4 @@ pub mod chain_spec;
 pub mod cli;
 pub mod node_rpc;
 pub mod service;
+use try_runtime_cli as _;
```

Comment:
Is it required or just missed here? Not featuregated which leads to compilation error.